README.md

```diff
@@ -21,6 +21,14 @@ git clone git@github.com:lagom/online-auction-java.git
 cd online-auction-java
 sbt runAll
 ```
+## Importing into IDEs
+
+### Intellij IDEA
+* **[Follow steps given here](http://www.lagomframework.com/documentation/1.3.x/java/IntellijSbtJava.html)**
+* Add **[Lombok](https://github.com/mplushnikov/lombok-intellij-plugin)** plugin by going to `File -> Settings -> Plugins` 
```

Comment:
See https://github.com/lagom/online-auction-java/issues/28#issuecomment-282752578

If you are fixing #28 here too, please mention that on your comments so that merging this PR closes both issues at once.